src/test/java/it/gov/pagopa/payhub/activities/activity/SendEmailIngestionFlowActivityTest.java

```diff
@@ -0,0 +1,462 @@
+package it.gov.pagopa.payhub.activities.activity;
+
+import it.gov.pagopa.payhub.activities.activity.paymentsreporting.SendEmailIngestionFlowActivity;
+import it.gov.pagopa.payhub.activities.activity.utility.SendEmailIngestionFlowActivityImpl;
+import it.gov.pagopa.payhub.activities.config.EmailTemplatesConfiguration;
+import it.gov.pagopa.payhub.activities.dao.IngestionFlowFileDao;
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.MailTo;
+import it.gov.pagopa.payhub.activities.dto.OrganizationDTO;
+import it.gov.pagopa.payhub.activities.dto.UserInfoDTO;
+import it.gov.pagopa.payhub.activities.service.OrganizationService;
+import it.gov.pagopa.payhub.activities.service.SendMailService;
+import it.gov.pagopa.payhub.activities.service.UserAuthorizationService;
+import jakarta.mail.MessagingException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.Optional;
+
+/**
+ * test class for send mail for ingestion activity
+ */
+@SpringBootTest(
```

Comment:
if possible, let's avoid @SpringBootTest test: to create a Spring context for each test could be heavy.
let's prefer to create real unit test, where you will create explicitly the object to be tested, injecting mocks through constructor (let's not change it this time, but remember it for next tests)